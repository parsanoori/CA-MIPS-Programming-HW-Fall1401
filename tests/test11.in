9
249612522
546620264
326841166
909587181
527251862
408214197
884281845
653433214
640928049