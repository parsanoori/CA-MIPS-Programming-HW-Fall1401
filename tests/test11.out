249612522
326841166
408214197
527251862
546620264
640928049
653433214
884281845
909587181